As we ship the tkey-runapp and -sign binaries as well as the signer app.bin
--- a/apps/signer/runsign.sh
+++ b/apps/signer/runsign.sh
@@ -21,5 +21,5 @@ fi
 msgf="$1"
 shift
 
-./tkey-runapp "$@" apps/signer/app.bin
-./tkey-sign "$@" "$msgf"
+tkey-runapp "$@" /usr/share/tillitis-key1-apps/signer/app.bin
+tkey-sign "$@" "$msgf"
